workspaces/src/error/mod.rs

```diff
@@ -0,0 +1,89 @@
+//! All errors that can occur within workspaces, including but not limited to
+//! the following: IO, RPC, Execution, Sandbox, DataConversion errors.
+
+mod impls;
+
+use std::borrow::Cow;
+
+/// A list specifying general categories of NEAR workspace error.
+#[derive(Clone, Debug, PartialEq, thiserror::Error)]
```

Comment:
Feels a bit weird to derive PartialEq, but not Eq. I think it's OK to commit to Eq here.